### DISCUSSION TITLE
Website Showcase Suggestions

### DISCUSSION THREAD

**Discussion by colinkiama** [nahuelwexd/vala-website#162] [Ideas]
~~While there's currently 6 in there. There is actually supposed to be 9.~~

Rather than showing the same 6 projects all the time, the website will display different projects in the showcase over time (Updated once a month or every few months maybe?).  This can be done by [editing this JSON file](https://github.com/vala-lang/vala-www/blob/dev/assets/showcase-items.json).

~~elementary OS and GNOME Boxes are definitely going to stay in there, the rest are placeholders (but can stay on there too).~~

Feel free to list down what you'd like to be on there.

**Comment by rakshit087** [comment]
Two more are:

* Gitg (https://wiki.gnome.org/Apps/Gitg)
* Connections (https://wiki.gnome.org/Apps/Connections)

They are written in Vala

**Comment by pervoj** [comment]
I wouldn't add Gitg, because it's almost not maintained.

What do you think about making some carousel from the showcase grid? We wouldn't have to limit to 6 or 9 items, we could add more items.

**Comment by colinkiama** [comment]
I'm not a big fan of carousels, they  can be tricky to create with a good experience across different input types (touch, keyboard, mouse,etc.), and even if we do make it work, there are going to be items that will be seen less because they are much further down the carousel.

This home page is supposed to summarise the language and the contents of the website. 

Maybe we should just stick 6 items and update them once or twice a month instead?
It would be super easy to do. [It's just this JSON file](https://github.com/colinkiama/vala-www/blob/dev/assets/showcase-items.json).

**Comment by pervoj** [comment]
Good idea

**Comment by colinkiama** [comment]
[GNOME Games](https://wiki.gnome.org/Apps/Games)

**Comment by ztefn** [comment]
Completely biased 😁️ showcase suggestion:
- Haguichi (https://haguichi.net)

**Comment by colinkiama** [comment]
[Timeshift](https://github.com/linuxmint/timeshift)

**Comment by colinkiama** [comment]
[Akira](https://github.com/akiraux/akira)

**Comment by RossComputerGuy** [comment]
[Genesis Shell](https://github.com/ExpidusOS/genesis), written in C++ and Vala but majority is Vala.

**Reply by colinkiama** [reply]
Hey, can you send me a screenshot to use please? 

I found an older one in Discord sent in January but I'm not sure how whether it has changed since then.

**Reply by RossComputerGuy** [reply]
![image](https://user-images.githubusercontent.com/19699320/190880207-c6d88991-cffc-45ea-b315-602e2bae8ee5.png)

This is the best I can get right now, I got it from a VM.

**Reply by colinkiama** [reply]
Thanks!

**Reply by RossComputerGuy** [reply]
However, it has not changed and I recommend using pictures from [this Tweet](https://twitter.com/ExpidusOS/status/1483510022294364161) as they are higher quality and show the shell working better.

**Reply by colinkiama** [reply]
Okay, will do

**Comment by colinkiama** [comment]
[TextSnatcher](https://github.com/RajSolai/TextSnatcher)

**Comment by lw64** [comment]
https://dino.im/ is a xmpp client written in vala.

**Comment by colinkiama** [comment]
[Tuba](https://github.com/GeopJr/Tuba)

**Comment by colinkiama** [comment]
[Workbench](https://apps.gnome.org/Workbench)

**Comment by colinkiama** [comment]
[Slick Greeter](https://github.com/linuxmint/slick-greeter)

**Comment by colinkiama** [comment]
[psequel](https://github.com/ppvan/psequel)
